src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -133,18 +133,6 @@ public bool MoveNext()
                 }
 
                 // Advance the state
-                _currentState = SearchState.SearchingBuiltinScripts;
```

Comment:
Is it safe to remove the enum number `SearchState.SearchingBuiltinScripts`? It doesn't seem to be needed anymore.

Reply:
Yes, looks safe to remove as it's no longer being used